Lib/gftools/push.py

```diff
@@ -98,11 +98,13 @@ def push_report(fp):
 def missing_paths(fp):
     paths = [fp.parent / p.path for p in parse_server_file(fp)]
     axis_files = [p for p in paths if p.name.endswith("textproto")]
+    axis_illustration_files = [p for p in paths if p.name.endswith("svg")]
```

Comment:
`endswith` can take a tuple, https://docs.python.org/3/library/stdtypes.html#str.endswith. This is cleaner:

```
axis_files = [p for p in paths if p.name.endswith(("textproto", "svg))]
```

Reply:
thank you! resolved